alembic/testing/util.py

```diff
@@ -78,3 +80,11 @@ def run_ddl(self):
         return fixture_functions.fixture(scope=ddl)(run_ddl)
 
     return decorate
+
+
+def ospath(linux_path):
+    """Takes a linux path with / and transforms it to the correct os path.
+    This is mainly to support windows \\"""
+    if win:
+        return normpath(linux_path)
```

Comment:
could we just be calling os.path.normpath in all cases, directly?    e.g. we don't need the conditional and then we don't really need "ospath".  just let normpath do it (or not).

Reply:
I'll look if it can be used. If had some problem with the leadit / on linux. But I can update the test to suppt it